### PR TITLE
[ci.yaml] Mac web_tool_tests force cache clear via 98.1

### PR DESCRIPTION
**Pull request by CaseyHillers**
https://github.com/flutter/flutter/issues/99372#issuecomment-1058531352

### CODE REVIEW

**Comment by CaseyHillers** [comment]
@godofredoc I see:

```
chillers@chillzone:~/flutter$ cipd describe flutter_internal/browsers/chrome/mac-amd64 -version BlBCql9ZngKYkn_tiqUUCnZKtZtbbCnsP0RgmLIn2ZwC
Package:       flutter_internal/browsers/chrome/mac-amd64
Instance ID:   BlBCql9ZngKYkn_tiqUUCnZKtZtbbCnsP0RgmLIn2ZwC
Registered by: user:godofredoc@google.com
Registered at: 2022-03-01 12:23:12.778879 -0800 PST
Refs:          none
Tags:
  version:98.1
  version:98
```

However, Mac web_tool_tests fails with:  `CIPD('ensure_installed') failed with: failed to resolve flutter_internal/browser-drivers/chrome/mac-amd64@version:98.1 (line 1): no such tag`

Is there another place I need to push the tag?

File: .ci.yaml [hunk 1]
```diff
@@ -3237,7 +3237,7 @@ targets:
       dependencies: >-
         [
           {"dependency": "android_sdk", "version": "version:31v8"},
-          {"dependency": "chrome_and_driver", "version": "version:98"},
+          {"dependency": "chrome_and_driver", "version": "version:98.1"},
```
Comment (godofredoc) [inline .ci.yaml:3240]: Can we update all the references of version:98 to version:98.1

**Comment by yjbanov** [comment]
This is another signal that we should stop using CIPD tags or refs. They are footguns. If we want to tag a version, it's trivial to use a named variable-like construct, such as:

```yaml
variables:
  chrome98: suCE89Njm7T4C2poqhfGK2Lgm3BBS7JL46e6wIv-WgMC
...
dependencies:
  chrome_and_driver: $chrome98
```

File: .ci.yaml (hunk 1 above)
Comment (CaseyHillers) [reply .ci.yaml:3240]: Done

**Comment by ditman** [comment]
Looks correctly tagged:

```
$ cipd search flutter_internal/browsers/chrome/mac-amd64 -tag version:98.1
Instances:
  flutter_internal/browsers/chrome/mac-amd64:BlBCql9ZngKYkn_tiqUUCnZKtZtbbCnsP0RgmLIn2ZwC
```

**Comment by CaseyHillers** [comment]
There was an issue in the retried build (it didn't pass 98.1). I filed https://github.com/flutter/flutter/issues/99523 as this is a bug in cocoon

**Review by godofredoc — APPROVED** [review]

**Pull request merged**